### ISSUE TITLE
[eglibc support] warning: failed finding version string

### ISSUE THREAD

**Issue by Arinerron** [io12/pwninit#8]
I am trying to use `pwninit --libc ./libc.so.6 --bin ./c` to fetch ld but am getting this error:
```yaml
bin: ./c
libc: ./libc.so.6

warning: failed finding version string
```

Is there an argument I can pass to be more verbose (so that I can get some debug information for you)? I am running `pwninit 2.1.1`.

**Comment by Arinerron** [comment]
`libc.so.6`: http://arinerron.com/files/libc.so.6

**Comment by io12** [comment]
Issue seems to be that this is eglibc (which ubuntu used to use) and not glibc. I will look into making the warning better and supporting eglibc.

**Comment by io12** [comment]
I released 2.1.4 which makes the warnings more descriptive, but eglibc support is harder, especially because the specific version (2.19-0ubuntu6.13) isn't on either of the mirrors pwninit uses. (Related: https://github.com/io12/pwninit/issues/5)

**Comment by io12** [comment]
All issues have been fixed in 2.2.0 :)
